packages/onboarding-ui/src/forms/RequestTrialForm/RequestTrialForm.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, Field, FieldGroup, TextInput } from '@rocket.chat/fuselage';
+import type { ReactElement } from 'react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import Form from '../../common/Form';
+
+type RequestTrialPayload = {
+  email: string;
+};
+
+type RequestTrialFormProps = {
+  onSubmit: SubmitHandler<RequestTrialPayload>;
+};
+
+const RequestTrialForm = ({
+  onSubmit,
+}: RequestTrialFormProps): ReactElement => {
+  const { t } = useTranslation();
+
+  const {
+    handleSubmit,
+    register,
+    formState: { isValidating, isSubmitting, isValid },
+    watch,
+  } = useForm<RequestTrialPayload>({ mode: 'onChange' });
+
+  watch('email');
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <FieldGroup>
+        <Field>
+          <Field.Label>
+            {t('form.requestTrialForm.emailAddr.label')}
+          </Field.Label>
+          <Field.Row>
+            <TextInput
+              {...register('email', { required: true })}
```

Comment:
Maybe passing a `validateEmail` callback as a prop would improve the customization of email validation here, as it was done in other forms.